src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -219,12 +232,22 @@ private void processDocs(final RootDoc rootDoc) {
     /**
      * @return the output extension to use, i.e., ".html" or ".php"
      */
-    protected String getOutputFileExtension() { return outputFileExtension; }
+    public String getOutputFileExtension() { return outputFileExtension; }
```

Comment:
I would like to have access to this paramater in my custom `DocWorkUnitHandler.getDestinationFilename()`, but with the previous implementation it is impossible unless the handler is in the same package as the Barclay one.

My use case is to generate a simpler name without the package and the same extension as the one provided in the command line. I also make public other getters because they may be useful for custom implementations: e.g., getIndexTemplateName will be useful to generate different indexes with the same template.

I will maintain this if you don't mind :)